api/controller.py

```diff
@@ -408,6 +408,8 @@ def load_lane(self, lane_identifier):
         if lane_identifier is None:
             # Return the top-level lane.
             lane = self.manager.top_level_lanes[library_id]
+            if isinstance(lane, Lane):
+                lane = self._db.merge(lane)
```

Comment:
I was getting some errors from the getting the parentage of a Lane that was detached from the session.

Reply:
I thought I had already made this change, and I did start it, but for whatever reason didn't finish. I bring this up because my incomplete change also handles the case where `lane` is a `WorkList` but not a `Lane`. In this case I wrote that all of the _children_ of the `Lane` also need to be merged. 

Since `top_level_lanes` can return a `WorkList` (if a site has multiple top-level lanes) I think that step is important here.

Reply:
Yeah, I double checked and I'm pretty sure we need this change. `load_lanes` expunges the children of a top-level `WorkList` so `load_lane` needs to merge the children back in.